modules/hcp-bm-hc.adoc

```diff
@@ -56,7 +56,7 @@ $ hcp create cluster agent \
 <6> Specify the etcd storage class name, for example, `lvm-storageclass`.
 <7> Specify the path to your SSH public key. The default file path is `~/.ssh/id_rsa.pub`.
 <8> Specify your hosted cluster namespace.
-<9> The default value for the control plane availability policy is `HighlyAvailable`.
+<9> Specify the availability policy for the hosted cluster components. Supported options are `SingleReplica` and `HighlyAvailable`. The default value is `HighlyAvailable`.
```

Comment:
nit: `s/hosted cluster/hosted control plane/` to align with the flag name and to distinguish it from `--infra-availability-policy` which is the availability policy of platform components running _within_ the hosted cluster.

Reply:
Good suggestion. I'll fix that. Thanks!